ctms/crud.py

```diff
@@ -477,38 +477,20 @@ def create_waitlist(
 ) -> Optional[Waitlist]:
     if waitlist.is_default():
         return None
-
-    # `create_waitlist` uses the `WaitlistInSchema``, which has a `subscribed`` property
-    # this makes sense because we're receiving this payload from Basket, and waitlists
-    # are just newsletters with naming conventions, so it has this property.
-    # Our Waitlist ORM model currently doesn't have an update property, so we need to remove it.
-    db_waitlist = Waitlist(email_id=email_id, **waitlist.dict(exclude={"subscribed"}))
+    db_waitlist = Waitlist(email_id=email_id, **waitlist.dict())
     db.add(db_waitlist)
     return db_waitlist
 
 
 def create_or_update_waitlists(
     db: Session, email_id: UUID4, waitlists: List[WaitlistInSchema]
 ):
-    # Remove waitlists that are marked as subscribed.
-    names_to_delete = [
-        waitlist.name for waitlist in waitlists if not waitlist.subscribed
-    ]
-    if names_to_delete:
-        db.query(Waitlist).filter(
-            Waitlist.email_id == email_id, Waitlist.name.in_(names_to_delete)
-        ).delete(
-            synchronize_session=False
-        )  # This doesn't need to be synchronized because the next query only alters the other remaining rows. They can happen in whatever order. If you plan to change what the rest of this function does, consider changing this as well!
-
     waitlists_to_upsert = [
-        UpdatedWaitlistInSchema(**waitlist.dict())
-        for waitlist in waitlists
-        if waitlist.subscribed
+        UpdatedWaitlistInSchema(**waitlist.dict()) for waitlist in waitlists
```

Comment:
Just something to note here, but we _really_ shouldn't need to do this `WaitlistInSchema` to `UpdatedWaitlistInSchema` conversion just to have the client generate an `update_timestamp`. Hopefully as we continue to untangle fixtures / complex tests, that will set us up to remove these `Updated.*Schema` schemas so that only the database is responsible for managing `update_timestamp` columns.

Reply:
👌 Deserves a follow-up